save-backend/src/test/kotlin/org/cqfn/save/backend/H2DatabaseTest.kt

```diff
@@ -0,0 +1,37 @@
+package org.cqfn.save.backend
+
+import org.cqfn.save.backend.entities.Billionaires
+import org.cqfn.save.backend.entities.TestEntity
+import org.cqfn.save.backend.repository.BillionairesRepository
+import org.cqfn.save.backend.repository.TestEntityRepository
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+
+
+@SpringBootTest(classes = [SaveApplication::class])
+class H2DatabaseTest {
+    @Autowired
+    private val testEntityRepository: TestEntityRepository? = null
```

Comment:
Does Autowired play okay with `val`?

Reply:
The test is passing, so I guess it's ok

Reply:
Does it need to be nullable then? Or maybe `lateinit`?

Reply:
Made it lateinit var